testassets/InteropTestsWebsite/Certs/ca.pem

```diff
@@ -0,0 +1,15 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Maybe we can use these certs for the examples as well. Less duplication and infrastructure.

Reply:
sg, let's address in a follow up PR?